src/main/java/bio/terra/workspace/service/resource/WsmResource.java

```diff
@@ -133,10 +133,13 @@ public boolean equals(Object o) {
 
     WsmResource that = (WsmResource) o;
 
-    if (workspaceId != null ? !workspaceId.equals(that.workspaceId) : that.workspaceId != null) return false;
-    if (resourceId != null ? !resourceId.equals(that.resourceId) : that.resourceId != null) return false;
+    if (workspaceId != null ? !workspaceId.equals(that.workspaceId) : that.workspaceId != null)
```

Comment:
Is there a way to invert this so we don't have so many negations?

Reply:
This is auto-generated by IntelliJ. We should talk about what code gen style we want to use and then use it everywhere. I agree this is ugly.